packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -1,4 +1,3 @@
-import { get } from 'http';
```

Comment:
This was a bug that was caught from this, I'd rather we merge this sooner rather than later.